immuni_analytics/models/exposure_data.py

```diff
@@ -69,6 +69,7 @@ class ExposurePayload(Document):
     """
 
     province = StringField(required=True)
+    symptoms_started_on = DateField(required=False)
```

Comment:
I'd add a comment about the `required=False`. To a new-reader, it would seem like the wrong value when the Exposure Ingestion will be consistently inserting such data as well.

What about something like the following?
```python
# NOTE: the field is marked as not required to support any data forwarded by the first version of the Exposure Ingestion Service, which did not include symptoms_started_on.
```